contracts/utils/deploy.js

```diff
@@ -622,7 +630,7 @@ const configureGovernanceContractDurations = async (reduceQueueTime) => {
     await setStorageAt(
       governorSix.address,
       "0x4",
-      "0x0000000000000000000000000000000000000000000000000000000000000001" // 1 block
+      "0x0000000000000000000000000000000000000000000000000000000000001C20" // 1 block
```

Comment:
The comment should probably say "7200 blocks"?

Reply:
fixed it in a later commit

Reply:
Yeah, that was my bad 😅